source/docs/software/basic-programming/git-getting-started.rst

```diff
@@ -0,0 +1,250 @@
+Git Version Control Introduction
+================================
+
+`Git <https://git-scm.com/about>`_ is a Distributed Version Control System (VCS) created by Linus Torvalds, also known for creating and maintaining the linux kernel. Version Control is a system for tracking changes of code for developers. The advantages of Git Version Control are:
+
+- Seperate testing environments into *branches*
+- Ability to navigate to a particular *commit* without removing history
+- Ability to manage *commits* in various ways, including combining them
+- Various other features, see `here <https://git-scm.com/about>`__
+
+Prerequisites
+-------------
+
+You have to download and install Git from the following links:
+
+- `Windows <https://git-scm.com/download/win>`_
+- `Mac OS X <https://git-scm.com/download/mac>`_
```

Comment:
Mac comes with git I believe, and if not, just typing `git` into terminal will prompt you to install Xcode command line tools, which installs git.

Reply:
Mac does not come with git, and the official installer is preferred in this case.